### PR TITLE
Create LlamaIndex documentation and Manual_QA.md

### PR DESCRIPTION
**Pull request by chandrevdw31**
## Description

Manual QA tests and documentation for LlamaIndex

**Fixes** #6889

## Type of change

(Please delete options that are not relevant)

- [ ] 🐛 Bug fix (non-breaking change which fixes an issue)
- [ ] ⚡ New feature (non-breaking change which adds functionality)
- [ ] 📢 Breaking change (fix or feature that would cause existing functionality not to work as expected)
- [x] 📄 This change requires a documentation update

## Verification Process

To ensure the changes are working as expected:

 - [ ]   Test Location: Specify the URL or path for testing.
 - [ ]   Verification Steps: Outline the steps or queries needed to validate the change. Include any data, configurations, or actions required to reproduce or see the new functionality.

## Additional Media:

- [x] I have attached a brief loom video or screenshots showcasing the new functionality or change.

## Checklist:

- [x] My code follows the style guidelines(PEP 8) of MindsDB.
- [x] I have appropriately commented on my code, especially in complex areas.
- [x] Necessary documentation updates are either made or tracked in issues.
- [ ] Relevant unit and integration tests are updated or added.

### CODE REVIEW

File: docs/integrations/ai-engines/llamaindex.mdx [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+
+On execution you get:
+
+<img src="/assets/tutorials/llamaindex/2.create_model.png" />
+
+The status of the model can be verified that it has trained successfully into a complete status by using the `DESCRIBE` syntax.
+
+```sql
+DESCRIBE qa_blackrock```
```
Comment (martyna-mindsdb) [inline docs/integrations/ai-engines/llamaindex.mdx:56]: Please move ``` to the next line, like this:

> \```sql
> DESCRIBE qa_blackrock
> \```

File: docs/integrations/ai-engines/llamaindex.mdx [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+FROM mindsdb.qa_blackrock
+WHERE Questions = 'What is the best long term investment with minimal risks for private investors';
+```
+
+On execution you get:
+
+<img src="/assets/tutorials/llamaindex/4.select_model.png" />
+
+Answer:
+`The best long term investment with minimal risks for private investors is a diversified portfolio of low-cost index funds. These funds typically track a broad market index, such as the S&P 500, and provide exposure to a wide variety of publicly traded companies with minimal risk. Blackrock is one of the largest asset management companies in the world, and offers a range of index funds, exchange-traded funds, and other investment products for private investors.`
```
Comment (martyna-mindsdb) [inline docs/integrations/ai-engines/llamaindex.mdx:72]: Please put this answer in a code block, like this:

> \```
> The best long term investment with minimal risks for private investors is a diversified portfolio of low-cost index funds. These funds typically track a broad market index, such as the S&P 500, and provide exposure to a wide variety of publicly traded companies with minimal risk. Blackrock is one of the largest asset management companies in the world, and offers a range of index funds, exchange-traded funds, and other investment products for private investors.
> \```

File: docs/integrations/ai-engines/llamaindex.mdx [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+  input_column = 'Questions',
+  openai_api_key = 'your_api_key';
+  ```
+
+Where:
+
+| Expression                             | Description                                |
+|----------------------------------------|--------------------------------------------| 
+| `qa_blackrock`                         | The name provided to the model.            |
+| `engine`                               | The engine used,which is Llamaindex.       |
```
Comment (martyna-mindsdb) [inline docs/integrations/ai-engines/llamaindex.mdx:42]: Please update description to: `The engine name - here, llamaindex.`

File: docs/integrations/ai-engines/llamaindex.mdx [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+| `index_class`                          | The vector store index used.               |
+| `reader`                               |The reader that reads the data/webpage.     |
```
Comment (martyna-mindsdb) [inline docs/integrations/ai-engines/llamaindex.mdx:44]: Please add `source_url_link` after `reader`. And description: `The webpage link.`

File: docs/assets/tutorials/llamaindex/.gitkeep [hunk 5]
```diff
@@ -0,0 +1 @@
+For llamaindex images
```
Comment (ZoranPandovski) [inline docs/assets/tutorials/llamaindex/.gitkeep:1]: Let's remove this file

**Review by ZoranPandovski — CHANGES_REQUESTED** [review]

File: docs/assets/tutorials/llamaindex/.gitkeep (hunk 5 above)
Comment (chandrevdw31) [reply docs/assets/tutorials/llamaindex/.gitkeep:1]: removed

**Review by martyna-mindsdb — APPROVED** [review]

**Review by ZoranPandovski — APPROVED** [review]

**Pull request merged**